includes/rest/class-nodeinfo.php

```diff
@@ -173,7 +173,7 @@ public static function discovery( $request ) {
 		$discovery['links'] = array(
 			array(
 				'rel' => 'http://nodeinfo.diaspora.software/ns/schema/2.0',
-				'href' => \get_rest_url( null, 'activitypub/1.0/nodeinfo' ),
+				'href' => get_rest_url_by_path( 'nodeinfo' ),
```

Comment:
`get_rest_url_by_path` is in the `Activitypub` namespace, so you have to add `use function Activitypub\get_rest_url_by_path` as the top of the class.